commands-yml/commands/interactions/touch/touch-perform.yml

```diff
@@ -0,0 +1,115 @@
+---
+name: Touch Perform
+short_description: Perform a touch action sequence
+   
+description:
+  |
+    This functionality is only available from within a mobile context
+
+    'Touch Perform' works similarly to the other singular touch interactions, except that this allows you to chain together more than one touch action as one
+    command. This is useful because Appium commands are sent over the network and there's latency between commands. This latency can make certain touch 
+    interactions impossible because some interactions need to be performed in one sequence. Vertical, for example, requires pressing down, moving to a different
+    y coordinate, and then releasing. For it to work, there can't be a delay between the interactions.
+
+example_usage:
+  java:
+    |
+      TouchAction action = new TouchAction(driver);
+      action.press(10, 10);
+      action.moveTo(10, 100);
+      action.release();
+      action.perform();
+  python:
+    |
+      actions = TouchAction(driver)
```

Comment:
It might be nice to have the import here
```python
from appium.webdriver.common.touch_action import TouchAction
# ...
action = TouchAction(driver)
```